flow/activities/flowable.go

```diff
@@ -779,8 +778,26 @@ func (a *FlowableActivity) ScheduledTasks(ctx context.Context) error {
 				updatedAt:  updatedAt,
 			}, nil
 		})
+		return infos, err
+	}
+
+	// final metrics recording before workflow exits (but don't sweat it if it fails)
+	defer func() {
+		infos, _ := queryFlowsInfo()
+		if infos == nil {
+			return
+		}
+		finalCtx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
+		defer cancel()
+		_ = a.RecordMetrics(finalCtx, infos)
+		_ = a.RecordSlotSizes(finalCtx, infos)
```

Comment:
```suggestion
		_ = a.RecordMetrics(finalCtx, infos)
```
don't think slot size info is that urgent